### PR TITLE
Updated gsoc.yml with prakharcode's feed

### PR DESCRIPTION
**Pull request by prakharcode**

### CODE REVIEW

**Review by dpshelio — CHANGES_REQUESTED** [review]
If you think you won't use it for other stuff then it's ok

File: gsoc.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ gsoc2018:
     rss_feed: "https://medium.com/feed/@swapsha96"
     project: 'TimeLab'
   prakharcode:
-    rss_feed: "FIXME"
+    rss_feed: "https://prakharcode.github.io/feed.xml"
```
Comment (dpshelio) [inline gsoc.yml]: @prakharcode would it be possible to get the feed only for the GSoC tag? You may want to write about other stuff in your blog.

**Comment by prakharcode** [comment]
@dpshelio changed the field to specific gsoc tag.
thanks to @oxinabox's code for tag-specific feed generation.

**Comment by oxinabox** [comment]
I stole the idea from someone else in turn :-)

**Review by dpshelio — APPROVED** [review]

File: gsoc.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ gsoc2018:
     rss_feed: "https://medium.com/feed/@swapsha96"
     project: 'TimeLab'
   prakharcode:
-    rss_feed: "FIXME"
+    rss_feed: "https://prakharcode.github.io/gsoc_feed.xml"
```
Comment (dpshelio) [inline gsoc.yml:24]: Thanks!

**Pull request merged**